### PR TITLE
Fixes #1379 Add since attribute

### LINKED ISSUE #1379: Proposal: Adding 'since' attribute to rules documenation
Proposing the addition of a 'since' attribute to be added to the front-matter of the rules documentation. This would be used to indicate since what version of eslint-plugin-vue a given rule exists. 

This proposal is in the context that not everyone has the latest version of eslint-plugin-vue (such as in a corporate environment), but may be looking at the documentation. It would give them a hint whether the rule was addd prior or after the version of eslint-plugin-vue being used in the project.

Example

```md
---
pageClass: rule-details
sidebarDepth: 0
title: vue/no-unused-properties
description: disallow unused properties
since: v7.1.0
---
...
```

### PR DESCRIPTION
**Pull request by ajmas**

### CODE REVIEW

**Review by ota-meshi — CHANGES_REQUESTED** [review]
Thank you for your lot of work!
Could you restore the blanks at the end of line that were intended for line breaks?

**Comment by ajmas** [comment]
I'll make the changes, though may be delayed slightly

File: docs/rules/README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 sidebarDepth: 0
 ---
+---
```
Comment (ota-meshi) [inline docs/rules/README.md:4]:
```suggestion
```

File: docs/rules/html-comment-indent.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 pageClass: rule-details
 sidebarDepth: 0
 title: vue/html-comment-indent
-description: enforce consistent indentation in HTML comments
+since: v7.0.0
+---
```
Comment (ota-meshi) [inline docs/rules/html-comment-indent.md:6]:
```suggestion
description: enforce consistent indentation in HTML comments
since: v7.0.0
```

**Review by ota-meshi — CHANGES_REQUESTED** [review]

**Review by ota-meshi — APPROVED** [review]
Thank you!

**Pull request merged**